dvc/utils/pkg.py

```diff
@@ -1,6 +1,15 @@
 from dvc.utils import is_binary
 
 
+def is_conda():
+    try:
+        from .build import PKG  # patched during conda package build
```

Comment:
When we start including this file into other packages, we would be able to simply delete this `dvc/utils/pkg.py` and have `build.py`(or maybe `pkg.py` if we decide to rename it) have `PKG = "something"` defined and then just `from .build import PKG` and use it instead of `get_package_manager`. That would look and feel very nice. :slightly_smiling_face:

Reply:
@efiop I will rework that method in the next PR where I include patches for other builds as well ;)